solana/ts/sdk/ntt.ts

```diff
@@ -0,0 +1,799 @@
+import { type ChainName, toChainId, coalesceChainId, type ChainId, SignedVaa } from '@certusone/wormhole-sdk'
+import { serializeLayout, toChainId as SDKv2toChainId } from "@wormhole-foundation/sdk-base";
+import { deserialize } from "@wormhole-foundation/sdk-definitions";
+//TODO these need to be properly exported by the SDK
+import { nttManagerMessageLayout, NttManagerMessage, nativeTokenTransferLayout } from "@wormhole-foundation/sdk-definitions/dist/cjs/payloads/ntt";
+import { derivePostedVaaKey, getWormholeDerivedAccounts } from '@certusone/wormhole-sdk/lib/cjs/solana/wormhole'
```

Comment:
these should also be available in the sdkv2

https://github.com/wormhole-foundation/connect-sdk/blob/5f530e25e974c51e078a34a710b765deac662082/platforms/solana/protocols/core/src/utils/accounts/postedVaa.ts#L4

https://github.com/wormhole-foundation/connect-sdk/blob/5f530e25e974c51e078a34a710b765deac662082/platforms/solana/protocols/core/src/utils/cpi.ts#L38

Reply:
yeah, I didn't try to purge the old sdk in its entirety, only replace the redundant payloads and combine overlapping functionality between ntt and quoter.